tests/test_BaseSolver.py

```diff
@@ -140,7 +140,7 @@ def test_comm_without_mpi(self):
         # initialize solver
         solver = SOLVER("testComm", comm=None)
         self.assertTrue(solver.comm is None)
-        solver.printCurrentOptions()
+        solver.printCurrentOptions()  # this should print current options twice since comm is not set
```

Comment:
To make a better connection I would add that this is specifically because N_PROCS = 2

Reply:
Right, yep